jekyll/_cci2/container-runner.adoc

```diff
@@ -95,6 +95,51 @@ agent:
           - name: "other"
 ```
 
+[#unsafe-retries]
+=== Unsafe retries
+
+Unsafe retries enable container runner to automatically rerun tasks if there are unexpected interruptions during their execution. These disruptions could be due to network connectivity issues, the underlying node shutting down, or other unpredictable causes. This feature can also be used when scheduling workloads on spot instances, which often come with cost-saving benefits at the risk of pod preemptions with many Kubernetes providers.
+
+[CAUTION]
+====
+This feature is called “unsafe retries” for a reason. Unlike automatic retries on startup, retrying tasks during runtime can be risky. This is because tasks can have arbitrary steps that produce side effects which are not idempotent or stateless. It is important to use unsafe retries knowing the impact it may have on jobs and workflows. Use this feature with jobs and workflows that are fault-tolerant.
+====
+
+Here is how unsafe retries work with container runner:
+
+1. If a pod fails or gets evicted during runtime, container runner will "un-claim", or release the task.
+2. All associated resources of the task are cleaned up.
```

Comment:
s/All associated resources/All CircleCI Self Hosted Runner cluster resources/

This could give the impression that we'd somehow cleanup generated data or state external to the task management.